libs/template/presets/storefront/experience/footer.ts

```diff
@@ -279,12 +280,7 @@ export const FooterTemplate: ExperienceComponent = {
       {
         layout:
           featureVersion >= '1.2'
-            ? {
-                type: 'flex',
-                divider: true,
-                bleed: true,
-                sticky: true,
-              }
+            ? { type: 'flex', bleed: true, sticky: true }
```

Comment:
Shouldn't we add a new condition?

Reply:
This was a bug and was also not having any effect. Hence i took the freedom to drop it.